lib/streamlit/git_util.py

```diff
@@ -23,13 +23,12 @@ def __init__(self, path):
 
             self.repo = git.Repo(path, search_parent_directories=True)
             self.git_version = self.repo.git.version_info
+
             if self.git_version >= MIN_GIT_VERSION:
                 git_root = self.repo.git.rev_parse("--show-toplevel")
                 self.module = os.path.relpath(path, git_root)
-
         except:
             # The git repo must be invalid for the following reasons:
-            #  * git binary or GitPython not installed
```

Comment:
Hm, why did we remove this comment -- is it no longer true?

Reply:
https://github.com/streamlit/streamlit/pull/2422#discussion_r538982834